src/main/java/frc/robot/Robot.java

```diff
@@ -25,15 +26,6 @@
   private String m_autoSelected;
   private final SendableChooser<String> m_chooser = new SendableChooser<>();
 
-  private PWMSparkMax leftMotorController1; 
-  private PWMSparkMax leftMotorController2; 
-  private SpeedControllerGroup leftSide; 
-
-  private PWMSparkMax rightMotorController1;
-  private PWMSparkMax rightMotorController2;
-  private SpeedControllerGroup rightSide;
-
-
   Drive drivetrain;
```

Comment:
I think this should be private

Reply:
Yea, that would be better practice for good encapsulation.

Reply:
doesn't allow for accessing in robotSetup for some reason

Reply:
That doesn't seem right, it should work/be allowed.